src/bundles/peer-locations.js

```diff
@@ -82,19 +82,40 @@ export default function (opts) {
     })
   )
 
-  bundle.selectPeerCoordinates = createSelector(
+  const COORDINATES_RADIUS = 8
```

Comment:
The map will feel bit empty on desktop, where users have more strict connection limits by default.

Quick fix: I propose a tweak to decrease range from 8 to 4:

```suggestion
  const COORDINATES_RADIUS = 4
```

### radius 8 (before)
![empty-Screenshot_2020-04-23 Peers - IPFS(1)](https://user-images.githubusercontent.com/157609/80099281-7f7cbf00-856e-11ea-9596-82bb087f9b6a.png)

### radius 4 (after)
![Screenshot_2020-04-23 Peers - IPFS](https://user-images.githubusercontent.com/157609/80099168-53613e00-856e-11ea-8da8-71a2b56c3a4f.png)

----

I believe we also should tweak the Index range to be optimized for ipfs-desktop users, which [will have between 50 and 300 peers](https://github.com/ipfs-shipyard/ipfs-desktop/blob/47e6306db586158a835dcf09cce6fa42279d8c5f/src/daemon/config.js#L46-L47).

Perhaps change the Index ranges to 1-10, 10-100, 100+ ?